ecom_test/tests/backend_tests/products/test_list_products_with_filter.py

```diff
@@ -0,0 +1,40 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.dao.products_dao import ProductsDAO
```

Comment:
I am not a fan of the name 'src_file'.
It is a direcotry but you are calling it 'file'.
Just call it 'src' that is very common and more professional.

Reply:
will take care of for the next commit